setup.py

```diff
@@ -17,18 +17,15 @@
         'Intended Audience :: Developers',
         'License :: OSI Approved :: BSD License',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
         'Topic :: Software Development :: Libraries :: Python Modules',
         'Development Status :: 5 - Production/Stable',
         'Operating System :: OS Independent',
     ],
     long_description=open('README.rst').read(),
     install_requires=[
-        "django-mptt>=0.8.6,<1.0",
+        "django-mptt>=0.11.0",
```

Comment:
I'd prefer to add `,<1.0` here, in case a future major version has backwards incompatible changes.